src/components/HookSubscriber/HookSubscriber.tsx

```diff
@@ -73,6 +74,7 @@ const HookSubscriber: FC = () => {
     subscribeManageTransactions();
     subscribeSavingsTransactions();
     subscribeWaitForBridgeStatus();
+    subscribeBridgeAllowances();
```

Comment:
you are already subscribing in hook file, so this can be removed.

Reply:
updated